integration_tests/ctesque/src/androidTest/java/android/app/ActivityInstrTest.java

```diff
@@ -113,4 +115,44 @@ public void audioManager_activityContextEnabled() {
       System.setProperty("robolectric.createActivityContexts", originalProperty);
     }
   }
+
+  @Test
+  public void accountManager_activityContextEnabled() {
+    String originalProperty = System.getProperty("robolectric.createActivityContexts", "");
+    System.setProperty("robolectric.createActivityContexts", "true");
+    try {
+      AccountManager applicationAccountManager =
+          (AccountManager)
+              ApplicationProvider.getApplicationContext().getSystemService(Context.ACCOUNT_SERVICE);
+      Log.d("Test", "Retrieved applicationAccountManager instance");
```

Comment:
Remove these log statements, they are not necessary. The assertions will check the values.